Assets/ZSSRichTextEditor.js

```diff
@@ -1064,7 +1065,9 @@ ZSSEditor.markVideoUploadDone = function(videoNodeIdentifier) {
     if (videoNode.parent().attr("id") == this.getVideoContainerIdentifier(videoNodeIdentifier)) {
         videoNode.parent().replaceWith(videoNode);
     }
-    videoNode.wrap(linkTag);
+    var joinedArguments = ZSSEditor.getJoinedFocusedFieldIdAndCaretArguments();
+    ZSSEditor.callback("callback-input", joinedArguments);
+    this.sendVideoReplacedCallback(videoNodeIdentifier);
```

Comment:
@SergioEstevao since these 3 lines are repeated above (1051-1053), is there a way we could simplify this?